unittests/webserver/ServerTest.cpp

```diff
@@ -313,7 +462,22 @@ TEST_F(WebServerTest, WsPayloadOverload)
 {
     std::string const s100(100, 'a');
     auto e = std::make_shared<EchoExecutor>();
-    auto const server = Server::make_HttpServer(cfg, ctx, std::nullopt, dosGuardOverload, e);
+    auto server = std::shared_ptr<Server::HttpServer<EchoExecutor>>();
```

Comment:
This entire block is repeated for each test.. if that's not a candidate for a helper function then i don't know what is.